src/webpack.js

```diff
@@ -160,20 +164,38 @@ const watchBundleChanges = (devMode, signingKey, callback, isForceClose) => {
       //retrieve the output of the compilation
       const data = stats.compilation.assets['index.js'].source();
 
+      const secretKey = Buffer.from(
+        signingKey.singingPrivateKey.toString(),
+        'hex',
+      );
+
       // update content data
       const dAppMetaData = getDappMetaData();
-      const content = JSON.stringify({
+      const content = {
         ...dAppMetaData,
         used_signing_key: signingKey.public_key,
         code: data,
-      });
+      };
+
+      const contentHash = hashDappContent(content);
```

Comment:
whats the definition of content? Please go over the PR and re think about naming of variables.